### DISCUSSION TITLE
not getting correct data.gro2lam file

### DISCUSSION THREAD

**Discussion by sanaper-cloud** [hernanchavezthielemann/GRO2LAM#56] [General]
Dear Hernan,
I want to convert simple ETHANOL gromacs file into lammps file but not getting the correct information  of atom types ,bond types, angle types etc in data.gro2lam file
I am also attaching here the all file.
[ETHANOL.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/8837578/ETHANOL.zip)

**Comment by hernanchavezthielemann** [comment]
Hello Sana @sanaper-cloud,

Could please try to explain what is the problem that you found?

Gro2lam generates a file with all the atom types in the forcefield that you input, but at the end just the ones assigned to atoms are used by lammps, same for all types. Maybe because of that you see an overpopulation of types.
I would like to implement a cleaner that makes the output neater, but since I would not expect a change in the results and I need some free time for that, this is not yet scheduled.

Best,
Hernan
